src/session.rs

```diff
@@ -1072,6 +1073,34 @@ impl Session {
         self.view_coords(self.views.active_id, p)
     }
 
+    /// Expand paths, e.g. by converting ~ into the user's home directory path
+    pub fn expand_path<P: AsRef<Path>>(&mut self, path: P) -> OsString {
+            let mut path = path.as_ref();
+
+            // for Linux and BSD and MacOS
+            if cfg!(unix) && path.starts_with("~") {
+
+                // ensure that the base directory list can be loaded by
+                // ensuring it has at least one element
```

Comment:
The comments are really helpful for code review, but I think we can remove them for merging, as the code is pretty clear in my opinion.

Reply:
I have since removed most of the comments.